### ISSUE TITLE
ppc_id has duplicate with different coordinates

### ISSUE THREAD

**Issue by monodera** [monodera/pfs_target_uploader#134]
**Describe the bug**
`ppc_id` in `ppc_*.ecsv` files have the same values with different coodinates

Example

```
ppc_code,ppc_ra,ppc_dec,ppc_pa,ppc_priority,Fiber usage fraction (%),ppc_resolution
Point_9,39.3048946884,-34.999945574,0.0,0.0001916966077603351,7.142857142857142,M
Point_9,39.92439643632,-34.52921765,0.0,3.2894209787803e-05,37.42690058479532,M
Point_9,40.121262958879996,-33.9334814455,0.0,0.0006662306820677229,2.2138680033416875,M
Point_9,40.504547647,-33.93046309,0.0,0.0005435269549591376,2.5898078529657473,M
Point_9,40.52126295888,-33.533481445499994,0.0,0.030437870668634633,0.04177109440267335,M
Point_9,40.52126295888,-33.9334814455,0.0,0.0010169906325617748,1.461988304093567,M
Point_9,40.52126295888,-33.9334814455,0.0,0.0022962491783339654,0.6683375104427736,M
Point_9,40.52126295888,-33.9334814455,0.0,0.007425871015696967,0.20885547201336674,M
```

**To Reproduce**
1. Run pointing simulation with some catalog (ideally clustered objects)
2. Click on 'Download' or 'Submit' then download the output.
3. Open 'ppc_*.ecsv' file
4. See `ppc_id`

**Expected behavior**
`ppc_id` is a unique identifier of pointings

**Additional context**
Maybe related to `group_id`?

**Comment by wanqqq31** [comment]
I tried to fix this bug. Now ppc_code is given by Point_[low/medium]_[i]. [i] is consistent with ppc_id on the plots. I guess there would be no such duplications, but please let me know if it still remains.

**Comment by monodera** [comment]
Thanks! It looks it's fixed. I'll close it once the branch is merged into the dev-main branch.
